src/reformatters/noaa/hrrr/forecast_48_hour/template_config.py

```diff
@@ -19,6 +26,16 @@
 INIT_TIME_FREQUENCY = pd.Timedelta("6h")
 LEAD_TIME_FREQUENCY = pd.Timedelta("1h")
 
+#  All Standard Cycles go to forecast hour 18
+#  Init cycles going to forecast hour 48 are 00, 06, 12, 18
+EXPECTED_FORECAST_LENGTH_BY_INIT_HOUR = pd.Series(
+    {
+        **{h: pd.Timedelta("18h") for h in range(0, 24 + 1)},
+        **{h: pd.Timedelta("48h") for h in range(0, 18 + 1, 6)},
```

Comment:
```suggestion
        **{h: pd.Timedelta("18h") for h in range(0, 24)},
        **{h: pd.Timedelta("48h") for h in range(0, 24, 6)},  # must be splatted last to overwrite 18h values
```
clever :) but just making sure we don't bit ourselves in a refactor later. also don't need an hour 24.